### ISSUE TITLE
makeTxDbFromGFF not working with GFF3File object

### ISSUE THREAD

**Issue by FelixErnst** [Bioconductor/GenomicFeatures#13]
This does not work:

```
gff <- GFF3File("sacCer3.gff")
makeTxDbFromGFF(gff)
```

This works
```
gff <- GFF3File("sacCer3.gff")
makeTxDbFromGFF(path(gff))
```
[sacCer3.zip](https://github.com/Bioconductor/GenomicFeatures/files/2786763/sacCer3.zip)

```
> sessionInfo()
R version 3.5.2 (2018-12-20)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=German_Germany.1252  LC_CTYPE=German_Germany.1252    LC_MONETARY=German_Germany.1252 LC_NUMERIC=C                    LC_TIME=German_Germany.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] Rsamtools_1.34.0            Biostrings_2.50.2           XVector_0.22.0              RPFTools_0.0.1              SummarizedExperiment_1.12.0 DelayedArray_0.8.0         
 [7] BiocParallel_1.16.5         matrixStats_0.54.0          GenomicFeatures_1.34.1      AnnotationDbi_1.44.0        Biobase_2.42.0              rtracklayer_1.42.1         
[13] GenomicRanges_1.34.0        GenomeInfoDb_1.18.1         IRanges_2.16.0              S4Vectors_0.20.1            BiocGenerics_0.28.0        

loaded via a namespace (and not attached):
  [1] assertive.base_0.0-7       colorspace_1.3-2           colorRamps_2.3             ggridges_0.5.1             htmlTable_1.13             base64enc_0.1-3           
  [7] rstudioapi_0.8             assertive.sets_0.0-3       ggrepel_0.8.0              bit64_0.9-7                assertive.data.uk_0.0-2    codetools_0.2-16          
 [13] splines_3.5.2              geneplotter_1.60.0         knitr_1.21                 SuppDists_1.1-9.4          Formula_1.2-3              assertive_0.3-5           
 [19] assertive.data.us_0.0-2    rJava_0.9-10               annotate_1.60.0            cluster_2.0.7-1            GO.db_3.7.0                graph_1.60.0              
 [25] compiler_3.5.2             httr_1.4.0                 backports_1.1.3            GOstats_2.48.0             assertthat_0.2.0           Matrix_1.2-15             
 [31] lazyeval_0.2.1             cli_1.0.1                  htmltools_0.3.6            acepack_1.4.1              prettyunits_1.0.2          tools_3.5.2               
 [37] bindrcpp_0.2.2             gtable_0.2.0               glue_1.3.0                 GenomeInfoDbData_1.2.0     Category_2.48.0            reshape2_1.4.3            
 [43] dplyr_0.7.8                Rcpp_1.0.0                 xtail_1.1.5                gdata_2.18.0               assertive.files_0.0-2      assertive.datetimes_0.0-2 
 [49] assertive.models_0.0-2     xfun_0.4                   stringr_1.3.1              gtools_3.8.1               XML_3.98-1.16              zlibbioc_1.28.0           
 [55] scales_1.0.0               hms_0.4.2                  RBGL_1.58.1                RColorBrewer_1.1-2         assertive.matrices_0.0-2   assertive.strings_0.0-3   
 [61] yaml_2.2.0                 memoise_1.1.0              RDAVIDWebService_1.20.0    gridExtra_2.3              ggplot2_3.1.0              UpSetR_1.3.3              
 [67] rpart_4.1-13               biomaRt_2.38.0             latticeExtra_0.6-28        stringi_1.2.4              RSQLite_2.1.1              genefilter_1.64.0         
 [73] checkmate_1.8.5            caTools_1.17.1.1           rlang_0.3.0.1              pkgconfig_2.0.2            bitops_1.0-6               lattice_0.20-38           
 [79] assertive.data_0.0-3       purrr_0.2.5                bindr_0.1.1                htmlwidgets_1.3            GenomicAlignments_1.18.1   assertive.properties_0.0-4
 [85] bit_1.1-14                 tidyselect_0.2.5           GSEABase_1.44.0            assertive.code_0.0-3       AnnotationForge_1.24.0     plyr_1.8.4                
 [91] magrittr_1.5               DESeq2_1.22.2              R6_2.3.0                   Hmisc_4.1-1                gplots_3.0.1               DBI_1.0.0                 
 [97] withr_2.1.2                foreign_0.8-71             pillar_1.3.1               assertive.numbers_0.0-2    nnet_7.3-12                survival_2.43-3           
[103] RCurl_1.95-4.11            tibble_1.4.2               crayon_1.3.4               assertive.types_0.0-3      KernSmooth_2.23-15         kSamples_1.2-8            
[109] progress_1.2.0             locfit_1.5-9.1             grid_3.5.2                 data.table_1.11.8          blob_1.1.1                 Rgraphviz_2.26.0          
[115] digest_0.6.18              xtable_1.8-3               munsell_0.5.0              assertive.reflection_0.0-4 sessioninfo_1.1.1      
```

**Comment by hpages** [comment]
Thanks Felix. This should be fixed in GenomicFeatures 1.34.2 (BioC 3.8) and GenomicFeatures 1.35.5 (BioC 3.9). Both will become available via `BiocManager::install()` in the next 24h or so.

**Comment by hpages** [comment]
I'm closing this. Feel free to reopen if I missed something.  H.
